inlong-agent/conf/agent.properties

```diff
@@ -112,14 +112,13 @@ agent.manager.auth.secretKey=
 
 
 ############################
-# prometheus config
+# metric config
 ############################
-# whether to enable prometheus
-agent.prometheus.enable=true
+metricDomains=Agent
+metricDomains.Agent.domainListeners=org.apache.inlong.agent.metrics.AgentPrometheusMetricHandler
```

Comment:
Would it be more intuitive using `xxxxListener` , such as `AgentPrometheusMetricListener`?